SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetricsDisplay.cpp

```diff
@@ -116,12 +129,15 @@ void ASpatialMetricsDisplay::DrawDebug(class UCanvas* Canvas, APlayerController*
 		DrawX = StatDisplayStartX + StatColumnOffsets[StatColumn_Worker];
 		Canvas->DrawText(RenderFont, FString::Printf(TEXT("%s"), *OneWorkerStats.WorkerName), DrawX, DrawY, 1.0f, 1.0f, FontRenderInfo);
 
-		DrawX += StatColumnOffsets[StatColumn_AverageFPS];
-		Canvas->DrawText(RenderFont, FString::Printf(TEXT("%.2f"), OneWorkerStats.AverageFPS), DrawX, DrawY, 1.0f, 1.0f, FontRenderInfo);
+		DrawX += StatColumnOffsets[StatColumn_AverageFrameTime];
+		Canvas->DrawText(RenderFont, FString::Printf(TEXT("%.2f ms"), 1000.f / OneWorkerStats.AverageFPS), DrawX, DrawY, 1.0f, 1.0f, FontRenderInfo);
 
 		DrawX += StatColumnOffsets[StatColumn_WorkerLoad];
```

Comment:
We should probably just remove load now as it's redundant (on NWX we use Load but with bUseFrameTimeAsLoad=true, which gives Average Frame Time under the Load column).  I'm not convinced the arbitrary load factor is very useful.